packages/babel-types/src/traverse/traverseFast.ts

```diff
@@ -5,22 +5,23 @@ import type * as t from "..";
  * A prefix AST traversal implementation meant for simple searching
  * and processing.
  */
-export default function traverseFast(
+export default function traverseFast<Options = {}>(
   node: t.Node | null | undefined,
-  enter: (node: t.Node, opts?: any) => void,
-  // todo(flow->ts) We could parametrize opts to T rather than any, so that the type is "forwarded" to the callback.
-  opts?: any,
+  enter: (node: t.Node, opts?: Options) => void,
+  opts?: Options,
 ): void {
   if (!node) return;
 
   const keys = VISITOR_KEYS[node.type];
   if (!keys) return;
 
-  opts = opts || {};
+  opts = opts || ({} as Options);
```

Comment:
Can we somehow enforce at the type level that Options is a type compatible with empty objects? Maybe `Options extends {}`?